suite-common/suite-services/package.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "@suite-common/suite-services",
+    "version": "1.0.0",
+    "private": true,
+    "license": "See LICENSE.md in repo root",
+    "sideEffects": false,
+    "main": "src/index",
+    "scripts": {
+        "lint": "eslint '**/*.{ts,tsx,js}'",
```

Comment:
`"lint:js": "g:eslint '**/*.{ts,tsx,js}'",`

Reply:
b82bbd915f3f1f04d2c0d125162dd5880e4ee3a1

Reply:
btw. it is `"lint:js": "yarn g:eslint '**/*.{ts,tsx,js}'",`, isn't it?